src/client/MapleCharacter.java

```diff
@@ -1695,15 +1692,16 @@ public void saveToDB(boolean dc, boolean fromcs) {
                 buddylist.setChanged(false);
             }
 
-            ps = con.prepareStatement("UPDATE accounts SET `nxCredit` = ?, `ACash` = ?, `mPoints` = ?, `points` = ?, `vpoints` = ?, `dpoints` = ?, `epoints` = ? WHERE id = ?");
+            ps = con.prepareStatement("UPDATE accounts SET `nxCredit` = ?, `ACash` = ?, `mPoints` = ?, `rPoints` = ?, `points` = ?, `vpoints` = ?, `dpoints` = ?, `epoints` = ? WHERE id = ?");
```

Comment:
Stuck with it for consistency, not too willing to do a rehaul such as that for now, though it'd be better to go back and completely revise them sometime later.